Src/Support/Google.Apis.Auth.Tests/OAuth2/ServiceAccountCredentialTests.cs

```diff
@@ -297,6 +328,66 @@ public async Task JwtCache_Expiry()
             Assert.NotSame(jwt0, jwt0Uncached);
         }
 
+        [Fact]
+        public async Task JwtScopes_Disabled_NoScopes_UseAud()
```

Comment:
This test and the next one can be a Theory, using [XUnit.Combinatorial](https://www.nuget.org/packages/Xunit.Combinatorial/) on a `useJwtAccessWithScopes` parameter.

And if you add a parameter for scopes, you can then have also the test for `Disabled_Scopes`. The Theory may be called JWT_UseAud, it would be covering the three cases in which aud is used.